lib/components/viewers/route-viewer.js

```diff
@@ -226,13 +316,19 @@ class RouteRow extends PureComponent {
             <b>{route.shortName}</b> {longName}
```

Comment:
This might be a question for the technical call. I would argue that even if using a different language the number and headsign should match the format used on the vehicle?